src/components/Form.js

```diff
@@ -139,7 +139,12 @@ export default class Form extends Component {
       return formData;
     }
 
-    return _pick(formData, fields);
+    let data = _pick(formData, fields);
+    if (Array.isArray(formData)) {
+      return Object.keys(data).map(key => data[key]);
```

Comment:
This won't work when formData is a non-empty array (for example, if formData is `["a","b","c"]`).

Reply:
Not sure what you mean. Suppose we have `formData = ["a","b","c"]` and `fields = ["0", "2"]`. Then, `data = _pick(formData, fields)` results in `data = {0: "a", 2: "c"}`. `Object.keys(data)` evaluates to `["0", "2"]`, and `["0", "2"].map(key => data[key])` evaluates to `[data["0"], data["2"]] = ["a", "c"]`, which is what we want.

Reply:
Oops, I thought for a second that fields would be an empty array. That makes sense, thanks!